figures/line.py

```diff
@@ -23,7 +23,8 @@ def _get_line(self, data, key):
         )
         p.line(x, y, legend_label="Progress", line_width=2)
         try:
-            p.y_range = Range1d(0, (int(max(y)) * 1.1))
+            if max(y) != "0":
+                p.y_range = Range1d(0, (int(max(y)) * 1.1))
```

Comment:
should we have an else case here? some defaulted value or something similar

Reply:
Shouldn't be necessary, we just want to make sure that we handle it different if it's 0. (Then we wont touch the axis ranges)

Reply:
My thought was that the else statement would be excessive and unneeded, but I might not see something